apps/golden-sample-app/src/environments/type.ts

```diff
@@ -5,4 +5,5 @@ export interface Environment {
   apiRoot: string;
   locales: string[];
   mockProviders?: Provider[];
+  common: any;
```

Comment:
Please, add type instead of `any`.
Also, type can be named descriptively, something like `sharedJourneyConfiguration`.

Reply:
done